randovania/games/samus_returns/logic_database/Surface - East.txt

```diff
@@ -136,9 +136,13 @@ Extra - asset_id: collision_camera_003
       All of the following:
           Morph Ball
           Any of the following:
-              Super Jump (Advanced) or Unmorph Extend (Intermediate) or Climb Rooms Vertically (High Jump)
-              # Freeze Enemy
-              Stand on Frozen Enemy (Beginner) and Fully Freeze Enemy
+              Super Jump (Beginner) or Climb Rooms Vertically (High Jump)
+              All of the following:
+                  Ice Beam
+                  Any of the following:
+                      # Short Freeze https://youtu.be/QJIMaZ8Qjpk
+                      Stand on Frozen Enemy (Intermediate)
+                      Charge Beam and Stand on Frozen Enemy (Beginner)
```

Comment:
using charge beam instead of the "fully freeze enemy" template means that this is not possible to freeze the enemy with melee. is this intentional?

Reply:
When testing this again, I couldn't figure out how the Melee worked, so i removed it for now

Reply:
lol